src/components/activities/manageBillActivity/ManageBillActivity.tsx

```diff
@@ -0,0 +1,288 @@
+
+  switch (activityTransitionState) {
+    case "TO_NEW_BILL":
+      return <Redirect to={`/bills`} />;
+    default:
+      return (
+        <div className="searchBills">
+          <AppHeader
+            userCredentials={userCredentials}
+            breadcrumbMap={breadcrumbMap}
+          />
+          <div className="searchBills__background">
+            <div className="container">
+              <div className="searchBills__panel">
+                <div
+                  className={classNames("searchBills__formData", {
+                    open_sidebar: isOpen,
+                  })}
+                >
+                  <div
+                    className="searchBills__formData__trigger_mobile"
+                    onClick={() => setIsOpen(!isOpen)}
+                  >
+                    {userCredentials?.displayName || "-"}
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      className="arrow_icon"
+                      data-name="Layer 1"
+                      viewBox="0 0 24 30"
+                      x="0px"
+                      y="0px"
+                    >
+                      <polygon points="12 17.02 4.08 9.1 5.5 7.69 12 14.19 18.5 7.69 19.92 9.1 12 17.02" />
+                    </svg>
+                  </div>
+                  <div className="searchBills__formData_sidebar">
+                    <div className="searchBills__formData_edit_button_wrapper">
+                      <div className="searchBills__formData_edit_button">
+                        <Button
+                          type="submit"
+                          variant="contained"
+                          color="primary"
+                          onClick={() =>
+                            setActivityTransitionState("TO_NEW_BILL")
+                          }
+                        >
+                          <Add fontSize="small" style={{ color: "white" }} />
+                          <span>{t("bill.newbill")}</span>
+                        </Button>
+                      </div>
+                    </div>
+                    <div className="searchBills__user_info">
+                      <Accordion expanded={expanded === "form"}>
+                        <AccordionSummary
+                          onClick={() => handleOnExpanded("form")}
+                        >
+                          <Search fontSize="small" style={{ color: "white" }} />
+                          <span> {t("bill.filter")}</span>
+                        </AccordionSummary>
+                        <AccordionDetails>
+                          <BillFilterForm
+                            className="searchBills__formData__item"
+                            onSubmit={submit}
+                            fields={initializeBillFilter(fromDate, toDate)}
+                          />
+                        </AccordionDetails>
+                      </Accordion>
+                      <Accordion expanded={true}>
```

Comment:
Should be: 

`expanded={expanded === "details"}>`

Reply:
Right. But I removed  the expanded, because user should see both sections without any action

Reply:
Ok, your point it's right. Maybe we can expand the Accordion logic in order to set a bunch of accordions open? 

Plus, in a future, I will wish to have a section called Billing data that takes all the billing information and it shows them inside some report as graphics, tables, ecc... but for now what you did it's OK.